### ISSUE TITLE
Feature set check may fail

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_SGP40#19]
Re this thread:
https://forums.adafruit.com/viewtopic.php?t=201980

Return isn't as expected so check fails.

Not finding any reference to the feature set command in any of the SGP40 refs.
https://github.com/adafruit/Adafruit_CircuitPython_SGP40/blob/4f872a90c581bbb22ebc623c7bd37eea051f9d60/adafruit_sgp40/__init__.py#L136-L141

Maybe just remove the check?

**Comment by caternuson** [comment]
Just received a new version of the SGP40 breakout to compare to an older version.

Reading the "feature set" register manually for each:

**OLDER BREAKOUT**
```python
Adafruit CircuitPython 7.3.3 on 2022-08-29; Adafruit QT Py RP2040 with rp2040
>>> import board
>>> i2c = board.STEMMA_I2C()
>>> i2c.try_lock()
True
>>> i2c.scan()
[89]
>>> buffer = bytearray(2)
>>> buffer[0] = 0x20
>>> buffer[1] = 0x2F
>>> i2c.writeto(89, buffer)
>>> i2c.readfrom_into(89, buffer)
>>> hex(buffer[0]), hex(buffer[1])
('0x32', '0x20')
>>>
```

**NEWER BREAKOUT**
```python
Adafruit CircuitPython 7.3.3 on 2022-08-29; Adafruit QT Py RP2040 with rp2040
>>> import board
>>> i2c = board.STEMMA_I2C()
>>> i2c.try_lock()
True
>>> i2c.scan()
[89]
>>> buffer = bytearray(2)
>>> buffer[0] = 0x20
>>> buffer[1] = 0x2F
>>> i2c.writeto(89, buffer)
>>> i2c.readfrom_into(89, buffer)
>>> hex(buffer[0]), hex(buffer[1])
('0x32', '0x40')
>>> 
```

So the masked added by #20 should be an adequate fix.

**Comment by caternuson** [comment]
Closing. Should be fixed by #20.

See also #21 for some additional fixes that were discovered at the same time and patched in that PR.
